packages/babel-plugin-transform-es2015-typeof-symbol/src/index.js

```diff
@@ -3,6 +3,14 @@ export default function ({ types: t }) {
 
   return {
     visitor: {
+      Scope({ scope }) {
+        if (!scope.getBinding("Symbol")) {
+          return;
+        }
+
+        scope.rename("Symbol");
```

Comment:
Valid point. If we want to only do Symbol for now, that's fine.

As @jdalton says, `Promise` is the one case that comes to mind. I don't think I'd say that's what _I_ expect imported `Promise` to be used by helpers, but it's possible people are using this behavior and assuming it is correct. I guess if we were going to change Promise, we'd at least want to do it in a non-patch release. It definitely seems like a terrible idea if anyone is relying on that, since their code would break immediately if they used `transform-runtime` or `external-helpers`, which you could potentially want to introduce at any time.